routes/event.js

```diff
@@ -222,12 +222,25 @@ async function generateSchedule(req, next, eventID, config, divisionOrder) {
 }
 
 router.get('/', async (req, res, next) => {
-	const season = await req.db.Season.getCurrent();
-
-	if (!season) {
-		return res.redirect(`/config/season?needsSeason=true&next=${req.originalUrl}`);
+	const selectedSeason = req.query.season;
```

Comment:
I'm unsure about this section - the flow works but is pretty much copied from the other place, so may be nicer to wrap in a function (possibly - not for definite)

My other thought is, if the season **does not** exist, do we want to be helpful and default to the current season, or helpful and tell them that the season they requested isn't set up?